lib/flappi.rb

```diff
@@ -9,7 +9,7 @@
 root_path = "#{File.dirname(__FILE__)}/flappi/"
 dependent_file_paths = DEPENDENT_FILES.map { |f| root_path + f }
 
-found_file_paths = Dir["#{root_path}**/*.rb"].sort
+found_file_paths = Dir["#{root_path}**/*.rb"]
```

Comment:
Due to [Lint/RedundantDirGlobSort](https://docs.rubocop.org/rubocop/cops_lint.html#lintredundantdirglobsort).